.gitignore

```diff
@@ -14,3 +14,8 @@ test/nashorn/lib
 NashornProfile.txt
 **/JTreport/**
 **/JTwork/**
+*.class
+# Eclipse specific folders
+.classpath
+.settings
+.project
```

Comment:
Just a note: changes to .gitignore does not belong in this PR. If you really intend to get this changes into the JDK (and not just as part of the Draft PR for development purposes), please file a separate issue for this.

Reply:
This will be definitely removed for the final PR. I would leave it here for a while because it prevents a bunch of junk being reported by `git status`.